### PR TITLE
Replace int3::asBP method

### PR DESCRIPTION
**Pull request by qubka**
For that pull: https://github.com/GregTech-Intergalactical/AntimatterAPI/pull/19

### CODE REVIEW

File: src/main/java/muramasa/gti/common/tileentities/multi/TileEntityPrimitiveBlastFurnace.java [hunk 1]
```diff
@@ -10,9 +10,9 @@
     public boolean onStructureFormed() {
         int3 controller = new int3(getPos(), getFacing());
         controller.back(1);
-        getWorld().setBlockState(controller.asBP(), Blocks.LAVA.getDefaultState(), 3);
+        getWorld().setBlockState(controller.toImmutable(), Blocks.LAVA.getDefaultState(), 3);
         controller.up(1);
-        getWorld().setBlockState(controller.asBP(), Blocks.LAVA.getDefaultState(), 3);
+        getWorld().setBlockState(controller.toImmutable(), Blocks.LAVA.getDefaultState(), 3);
```
Comment (Rongmario) [inline src/main/java/muramasa/gti/common/tileentities/multi/TileEntityPrimitiveBlastFurnace.java]: Can you commit this in a separate branch and open a separate PR?

Comment (qubka) [reply src/main/java/muramasa/gti/common/tileentities/multi/TileEntityPrimitiveBlastFurnace.java]: Can you explain more detailed? I see only one branch for Gregtech. Will I have access to do pull in a new branch?

Comment (Rongmario) [reply src/main/java/muramasa/gti/common/tileentities/multi/TileEntityPrimitiveBlastFurnace.java]: No... _you_ create a new branch, and PR into our _master branch_

Comment (qubka) [reply src/main/java/muramasa/gti/common/tileentities/multi/TileEntityPrimitiveBlastFurnace.java]: Ok, then close this PR. Is I should create a new branch for all  3 projects to commit?

Comment (Rongmario) [reply src/main/java/muramasa/gti/common/tileentities/multi/TileEntityPrimitiveBlastFurnace.java]: No, just revert the last commit, the one where you commited the lines I highlighted here. Leave this open as this is something separate, then commit these lines in another branch and open a separate PR.
